### DISCUSSION TITLE
Unclear Instructions

### DISCUSSION THREAD

**Discussion by A01378939** [elblogbruno/NotionAI-MyMind#15] [Q&A]
Hi Bruno, I just downloaded your extension and there are just soo many things to do to use the extension that I think it would be awesome for you to make a video tutorial.

I downloaded the exe installer and when running it, it asks me: "Which port you'd like to run the server on:". You never mention any port for what I read.

When clicking on "Options" by right-clicking the extension icon, it gets me to a page where I'm not sure if I should enter my IP address or the url of my notion page. 

Could you help me here? Thanks

**Comment by elblogbruno** [comment]
Hi!
I am so sorry for the problems you are having. I am iterating every day to try get the best way of having an easier setup or process. In every iteration I am learning from your feedback, and in the next release this is one of the things I want to attack!

First answer, the port is a number that identifies the server running. It can either be 8080,80 whatever. It exists because your IP or the computers IP running the Notion AI my Mind server may have other servers or programs running and the chrome extension and phone app may get lost knowing which server to contact. 

On the  extension you should enter the IP address of the server running.  This makes the extension aware of what server to contact. The notion URL should be set on the server settings you can access by accessing the server IP or address in your favourite browser or right click extension -> open server  button once you have set it correctly on the extension options.

If you may have more questions don't mind asking! 
I am all 👂!
Cordially
Bruno

**Reply by A01378939** [reply]
Ok, I think I'm done with that. How do I know the extension is working? When I click on it it doesn't do anything, right clicking it and choosing open my mind brings me to a "Your file couldn't be accessed" and choosing open server brings me to my Notion Page. Choosing refresh collections tells me "Successfully refreshed your collections". 

Am I missing something? :):

**Reply by A01378939** [reply]
Welp, I just can't work it through. Such a shame bc I see potential in this extension. Maybe if you had 10 spare min so we could reunite on zoom or sth like that to help me out I'd thank you a lot, but in the meantime I may be sticking w another Notion web clipper ):

**Comment by elblogbruno** [comment]
Hi!
Good morning!
Have you created a notion page like this with collections on it?

https://github.com/elblogbruno/NotionAI-MyMind/wiki/Notion-AI-My-Mind-Collections

This is the notion URL you should tell the server to get info from. Then on the browser extension you will see a button call refresh collections.

When you are on a website or select a piece of text for example , when you click to add content a popup should appear allowing you to choose where you want to save that content!

Before you could only had one collection this way , you can have multiple collections of different themes!

**Reply by elblogbruno** [reply]
https://github.com/elblogbruno/NotionAI-MyMind/wiki/I-have-installed-the-server,-what-to-do-next
Have you also see this?

When you install the server for the first time, no matter which way, the browser opens automatically with the page settings, in case at first don't know what is the IP or address of the server.

**Comment by A01378939** [comment]
I already got over all of that, but I still have some doubts.

Does the server exe has to be kept running? Or may I close it?

When I click Refresh collections no error nor success pop up advise comes up):

**Reply by elblogbruno** [reply]
Sure the server must keep running! The server is the one that contacts notion and AI tags your images and websites!

For the other part of the question have you created a structure of collections inside notion?

https://github.com/elblogbruno/NotionAI-MyMind/wiki/Notion-AI-My-Mind-Collections

Also if you can access the folder were the server is there is an app.log file you might share here and maybe we can see the problem happening on the log!

Cordially
Bruno

**Comment by elblogbruno** [comment]
https://github.com/elblogbruno/NotionAI-MyMind/wiki/Installing-the-Notion-AI-My-Mind-Server#installation-video-tutorial I have published a video tutorial on youtube!
